vcfconcat.c

```diff
@@ -549,6 +553,114 @@ static void concat(args_t *args)
     }
 }
 
+BGZF *hts_get_bgzfp(htsFile *fp);
+static void naive_concat(args_t *args)
+{
+    // only compressed BCF atm
+
+    BGZF *bgzf_out;
+    if ( strcmp("-",args->output_fname) )
+        bgzf_out = bgzf_open(args->output_fname,"w");
+    else
+        bgzf_out = bgzf_dopen(fileno(stdout), "w");
```

Comment:
No, unhelpful for Windows.  `bgzf_open` understands `"-"`, so these five lines can be one.